test/verify/check-shell-keys

```diff
@@ -34,6 +34,9 @@ class TestKeys(testlib.MachineCase):
         m.execute("useradd user -s /bin/bash -m -c User")
         m.execute("echo user:foobar | chpasswd")
 
+        if self.machine.image.startswith("rhel-8") or self.machine.image.startswith("centos-8"):
+            self.allow_journal_messages('passwd: bad argument --all: unknown option')
```

Comment:
and this really never should go into the journal. It's not something the admin did wrong.